scripts/upgrade_configuration.py

```diff
@@ -0,0 +1,184 @@
+"""
```

Comment:
Inconsistent docstring formatting

Reply:
Ok thanks it will be corrected

Reply:
I don't think we have any specific ones necessarily, they just need to be consistent throughout. Pick whichever one you prefer and can be easily made consistent